src/ensembl/src/shared/data/centromeres.ts

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
Is this file going to be shared across apps?

Reply:
Yes, any application that wants to draw human chromosomes with centromeres in them (EntityViewer being the most likely candidate) will have to use this file (until ENSWBSITES-385 is completed and we start getting this data from the backend).